src/I18n/DateFormatTrait.php

```diff
@@ -190,7 +190,7 @@ public function i18nFormat($format = null, $timezone = null, $locale = null)
         if ($timezone) {
             // Handle the immutable and mutable object cases.
             $time = clone $this;
-            $time = $time->timezone($timezone);
+            $time = $time->setTimezone($timezone);
```

Comment:
If we change the name then the constraints for chronos should be updated to the minor in which this method was added (assuming setTimezone() doesn't exist in 2.2).

Reply:
`setTimezone()` does exist in [Chronos 2.2](https://github.com/cakephp/chronos/blob/2.2.0/src/Traits/TimezoneTrait.php#L55)

Reply:
Great, so no other change needed.